src/App/Order/Model/PdkOrderLine.php

```diff
@@ -43,5 +47,6 @@ public function __construct(?array $data = null)
     {
         parent::__construct($data);
         $this->calculateVatTotals();
+        $this->settings = $this->product->getMergedSettings();
```

Comment:
Waarom is dit nu als losse property? Kan toch nog steeds in het product zelf onder `product->mergedSettings` oid?

Reply:
Vanwege `mergeProductSettings` in `ShipmentOptionsService`, en misschien ook voor later gebruik, kunnen we de orderLines makkelijk processen zonder een product te hoeven gebruiken om daar `mergedSettings` op te halen, dacht ik.

Reply:
Ik ga ’t aanpassen.